### PR TITLE
Add Grok deprecation release note

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request (PR) adds a release note announcing deprecation of Grok. It also removes documentation of Grok.

Issue number: None. PR made at the request of Sourabh Bhosale. 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: blog-cse/2023-08-22-application-update.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Deprecation Notice
+
+After careful evaluation, we have determined that Grok patterns are deprecated immediately for customers not using the feature in the last 30 days. A more robust and configurable solution is already available for customers in the Sumo Logic parsers. More details on how parsing works in Sumo Logic can be found in the [Parsing Language Reference Guide](/docs/cse/schema/parsing-language-reference-guide).
```
Comment (kimsauce) [inline blog-cse/2023-08-22-application-update.md:17]:
```suggestion
After careful evaluation, we have deprecated Grok patterns immediately for customers who've not used the feature in the last 30 days. Our more robust and configurable solution is already available for customers in the Sumo Logic parsers. More details on how parsing works in Sumo Logic can be found in the [Parsing Language Reference Guide](/docs/cse/schema/parsing-language-reference-guide).
```

File: blog-cse/2023-08-22-application-update.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+For customers who are still using Grok, further communication along with a path to migration to the Sumo Logic parsers will be provided in the coming weeks.
```
Comment (kimsauce) [inline blog-cse/2023-08-22-application-update.md:19]:
```suggestion
For customers who are still using Grok, further communication along with a path to migrate to the Sumo Logic parsers will be provided in the coming weeks.
```

**Review by kimsauce — APPROVED** [review]
Approved w/two comments @jpipkin1

**Comment by jpipkin1** [comment]
Thanks for the review, @kimsauce!

**Pull request merged**